CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Update the README.md with details of changes to the interface, this includes new environment 
```

Comment:
This line also seems out of place in a Go project. Port exposure should be irrelevant here, and "container parameters" as well.